dev-scheme/racket/racket-8.2-r1.ebuild

```diff
@@ -19,12 +19,14 @@ LICENSE="
 	chez? ( Apache-2.0 )
 	!chez? ( LGPL-3 )
 "
-# Bytecode generated by Racket is not compatible between versions
+# Bytecode generated by Racket is not compatible between versions.
+# In this case PV == bytecode version,
+# but for _patch, _pre, etc. it has to be set manually.
 SLOT="0/${PV}"
 KEYWORDS="amd64 ~arm ~ppc ~ppc64 x86"
-IUSE="X +chez +doc +futures +jit minimal +places +readline +threads"
+IUSE="X +chez +doc +futures +jit minimal +places +threads"
 # See bug #809785 re chez/threads
-REQUIRED_USE="futures? ( jit ) chez? ( threads )"
+REQUIRED_USE="futures? ( jit threads ) chez? ( threads ) places? ( threads )"
```

Comment:
Link to any relevant upstream commits or bugs in the bug / here, if there are any.

Reply:
afair it fails on configure if you do `USE="-threads places futures"`

Reply:
![image](https://user-images.githubusercontent.com/47485207/143725805-855604b3-b468-42d3-ae08-61395f6d98bf.png)

Reply:
Ah, I was just saying that I thought you'd mentioned some work upstream, so we should mention it in the Gentoo bug if you have any links.

Reply:
the one we talked before was about `chez` + `-threads`, threads for futures/places are probably to stay